app/lib/primer/tabbed_component_helper.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Primer
+  # Helper to share tab validation logic between components.
+  # The component will raise an error if there are 0 or 2+ selected tabs.
+  module TabbedComponentHelper
+    extend ActiveSupport::Concern
+
+    included do
+      include ViewComponent::SlotableV2
```

Comment:
If we update do `view_component` `v2.28.0`, this will no longer be needed 😄

Reply:
Yes!
But at the same time, the gem technically supports `view_component >= 2.0 && < 3`
Removing `include ViewComponent::SlotableV2` would drop support for anything `< 2.28`

What we could do is move this include to `Primer::Component` which would make things easier to migrate or even only include according to the `view_component` version

Reply:
> What we could do is move this include to Primer::Component which would make things easier to migrate or even only include according to the view_component version.

Let's do that. Maybe we could couple that with a tracking issue on this repo for things to do once we upgrade to VC v3?